apps/public-docsite-v9/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Fluent UI React v9 documentation",
   "scripts": {
     "build": "just-scripts build",
-    "build-storybook": "build-storybook -o ./dist/storybook --docs && node ./.storybook/fix-title.js 'Fluent UI React v9' ../dist",
+    "build-storybook": "build-storybook --no-manager-cache -o ./dist/storybook --docs && node ./.storybook/fix-title.js 'Fluent UI React v9' ../dist",
```

Comment:
align with other `build-storybook` task ( not caching manager should help exploit potential issues like this one with addon processing bug )